### DISCUSSION TITLE
adding a section to set maximum RAM and CPU access

### DISCUSSION THREAD

**Discussion by alirezach** [posit-dev/positron#5396] [Feedback] [answered]
Hello and thank you for your good job.
I have a suggestion to add an option to add maximum access to RAM and CPU in the Positron settings to increase the processing speed.

**Comment by juliasilge** [comment, marked as answer]
I may not quite be understanding your question. Are you wanting to limit the RAM and CPU that Positron has access to use? Or you are observing that Positron is not using available RAM and/or CPU? It would be helpful if you could share a few more details about the problem you are experiencing.

Looks like there is some related discussion in https://github.com/microsoft/vscode/issues/175792, and there are [other issues upstream you can see by searching "max memory"](https://github.com/search?q=repo%3Amicrosoft%2Fvscode+max+memory&type=issues).
